torch/ao/quantization/fx/_decomposed.py

```diff
@@ -0,0 +1,52 @@
+import torch
+from torch.library import Library, impl
+
+# Note: decomposed means decomposed quantized tensor, using decomposed so that the
+# name is not too long
+quantized_decomposed_lib = Library("quantized_decomposed", "DEF")
+
+quantized_decomposed_lib.define(
+    "quantize_per_tensor(Tensor input, float scale, int zero_point, int quant_min, int quant_max, ScalarType dtype) -> Tensor")
```

Comment:
So we need to register exactly the same schema somewhere into edge runtime. It seems a bit hard to maintain for current infra. If we change schema here we need to change schema in edge runtime otherwise it won't work.

Reply:
so should we define these in native_functions.yaml?